app/components/primer/button_component.rb

```diff
@@ -6,11 +6,15 @@ class ButtonComponent < Primer::Component
     status :beta
 
     DEFAULT_SCHEME = :default
+    LINK_SCHEME = :link
     SCHEME_MAPPINGS = {
       DEFAULT_SCHEME => "",
       :primary => "btn-primary",
       :danger => "btn-danger",
-      :outline => "btn-outline"
+      :outline => "btn-outline",
+      :block => "btn-block",
```

Comment:
Is it possible to have a button that is both `primary` and `block`? What about `outline` and `primary`?

Reply:
oh that's true, `block` is just a modifier, going to move it to a boolean
`outline` though is a scheme and shouldn't be used with another schemes